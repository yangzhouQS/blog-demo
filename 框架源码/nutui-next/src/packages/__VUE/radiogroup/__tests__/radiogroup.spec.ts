import { mount } from '@vue/test-utils';
import Radiogroup from '../index.vue';

test('prop modelValue', () => {
  const wrapper = mount(Radiogroup, {
    props: {
      modelValue: '1'
    }
  });
  expect(wrapper.html()).toMatchSnapshot();
});
