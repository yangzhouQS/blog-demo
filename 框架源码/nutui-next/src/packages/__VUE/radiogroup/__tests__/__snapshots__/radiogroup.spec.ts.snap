// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prop modelValue 1`] = `"<view class=\\"nut-radiogroup nut-radiogroup--vertical\\"></view>"`;
